internal/cli/dumpconfig.go

```diff
@@ -69,14 +60,9 @@ func (c *DumpconfigCommand) Run(args []string) int {
 	userConfig.Gpo.IgnorePriceRaw = userConfig.Gpo.IgnorePrice.String()
 	userConfig.Cache.RejournalRaw = userConfig.Cache.Rejournal.String()
 
-	// Currently, the configurations (userConfig) is exported into `toml` file format.
-	out, err := tomlSettings.Marshal(&userConfig)
-	if err != nil {
-		c.UI.Error(err.Error())
-		return 1
+	if err := toml.NewEncoder(os.Stdout).Encode(userConfig); err != nil {
+		log.Fatal(err)
```

Comment:
Can we use `c.UI.Error(err.Error())` here instead of log.Fatal? And also return 1?